<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CrowdEstimate</class>
 <widget class="QDialog" name="CrowdEstimate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>人群密度估计参数</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>背景建模方案</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="currentText">
    <string>MOG</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>MOG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>MOG2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OCL_MOG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>OCL_MOG2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUDA_MOG</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>CUDA_MOG2</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>密度分类模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>70</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>四级</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>百分比</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>自学习帧数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>130</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>140</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>210</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>历史帧数</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>150</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>120</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>130</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>140</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>160</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>170</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>180</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>190</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>210</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>220</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>240</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>250</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>260</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>270</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>280</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>290</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
